src/coreclr/src/inc/corinfo.h

```diff
@@ -955,6 +955,7 @@ enum CorInfoIntrinsics
     CORINFO_INTRINSIC_Span_GetItem,
     CORINFO_INTRINSIC_ReadOnlySpan_GetItem,
     CORINFO_INTRINSIC_GetRawHandle,
+    CORINFO_INTRINSIC_GetIsSupported,       // for interpreting hardware intrinsics
```

Comment:
This is legacy method for identifying intrinsics. It would be best to avoid adding more to it.

The preferred method for itentifying intrinsics is by name (e.g. look for `CORINFO_FLG_JIT_INTRINSIC` and related code).

Reply:
Hi @jkotas, thanks for pointing this out! I removed my changes in commit https://github.com/dotnet/runtime/pull/37825/commits/46a004c71e0e8e7614b160e702e4f636cf203774. However, I feel that a possibly more appropriate location to identify intrinsics is the handling of "intrinsics-like" methods after the `switch` for intrinsics:

https://github.com/dotnet/runtime/blob/46a004c71e0e8e7614b160e702e4f636cf203774/src/coreclr/src/vm/interpreter.cpp#L9074

Could you please check if this aligns with the pattern in the codebase? Thanks very much!

Reply:
This needs TODO comment that it is workaround that does not produce correct results. (more context in https://github.com/dotnet/runtime/pull/37825#issuecomment-643630137).

Reply:
Done in commit https://github.com/dotnet/runtime/pull/37825/commits/2fd5c4b610c0b52374403acae872fd0a6a18099e.